recipe/gen_patch_json.py

```diff
@@ -1421,6 +1421,14 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         # conda-libmamba-solver 22.3.1 and prior are not compatible
         if record_name == "conda-libmamba-solver" and record.get("timestamp", 0) <= 1650455037727:
             _replace_pin("libmambapy >=0.22", "libmambapy 0.22.*", record["depends"], record)
+
+        # typing-extensions 4.2.0 requires python >=3.7. Build 0 incorrectly specified >=3.6. Fixed in
+        # https://github.com/conda-forge/typing_extensions-feedstock/pull/30
+        if record_name in ["typing-extensions", "typing_extensions"]:
```

Comment:
`typing-extensions` doesn't depend on Python:
```suggestion
        if record_name == "typing_extensions":
```